shims/emr700/driver/src/main/java/org/pentaho/hadoop/shim/HadoopShim.java

```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho
  *
- * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ * Copyright (C) 2024-2025 by Hitachi Vantara, LLC : http://www.pentaho.com
```

Comment:
We aren't changing the license header dates at this time

Reply:
fixed